### PR TITLE
[12.0][FIX] hr_employee_calendar_planning: Force to auto generate calendar even when calendar is not active.

### PR DESCRIPTION
**Pull request by victoralmau**
Force to auto generate calendar even when calendar is not active + Migration script.

Please @pedrobaeza and @chienandalu can you review it?

@Tecnativa TT31606

### CODE REVIEW

File: hr_employee_calendar_planning/migrations/12.0.1.1.0/post-migration.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE resource_calendar
+        SET auto_generate = true
+        WHERE active = false""",
```
Comment (chienandalu) [inline hr_employee_calendar_planning/migrations/12.0.1.1.0/post-migration.py:13]: Should we ensure it better filtering those with the name like   'Auto generated calendar for employee'

Comment (pedrobaeza) [reply hr_employee_calendar_planning/migrations/12.0.1.1.0/post-migration.py:13]: It can't be due to the translations.

**Review by pedrobaeza — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]
Ok :+1:

**Pull request merged**